### PR TITLE
Pagination fixes

### PR DESCRIPTION
**Pull request by kemsguy7**
https://github.com/ChallengeForge/TheWeeklyChallenge/assets/31693594/8701f4b9-e4e3-4f32-b2b6-af85b21094d9

## Description

In this PR, I updated the team, guest and champions page to take same layout and card limiting functionality

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Changes

Please which part of the project your changes affect ?
- [ ] Frontend (Templates, CSS, JS)
- [ ] Backend (Dancer2, api-docs, Dockerfile)

### CODE REVIEW

**Comment by kemsguy7** [comment]
@manwar I've reverted the champions page in the main branch . Please kindly Review

File: prototype/Frontend/index.html
Comment (himanshipikachu) [inline prototype/Frontend/index.html:1]: Paths here haven't been changed @kemsguy7 , please remove these from your PR

File: prototype/Frontend/guest.html [hunk 1]
```diff
@@ -55,7 +54,7 @@
 
     <!-- Template Main CSS File -->
     <link href="/public/css/index.css" rel="stylesheet" />
-    <link rel="stylesheet" href="/public/css/teamStyle.css" />
+    <link rel="stylesheet" href="../../public/css/teamStyle.css" />
```
Comment (himanshipikachu) [inline prototype/Frontend/guest.html]: again path here @kemsguy7

File: prototype/Frontend/champions.html
Comment (himanshipikachu) [inline prototype/Frontend/champions.html:1]: No changes required for this file

File: public/css/teamStyle.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import '/public/css/index.css';
+@import '../../public/css/index.css';
```
Comment (himanshipikachu) [inline public/css/teamStyle.css]: path here too

**Comment by manwar** [comment]
@kemsguy7 please share short video how it looks now after your last change.

**Comment by kemsguy7** [comment]
https://github.com/ChallengeForge/TheWeeklyChallenge/assets/31693594/030a7e03-a719-4c92-95ec-28fdf6b3d5da


@manwar , please take a look, I've here's what the page looks like currently

**Review by manwar — COMMENTED** [review]
Looks good, thanks.

**Comment by manwar** [comment]
Thanks for your contributions. @kemsguy7

**Comment by kemsguy7** [comment]
> Thanks for your contributions. @kemsguy7

You're welcome mentor

**Pull request merged**